packages/instant/src/util/buy_quote_fetcher.ts

```diff
@@ -0,0 +1,73 @@
+// TODO: rename file and export object
+import { AssetBuyer, AssetBuyerError, BuyQuote } from '@0x/asset-buyer';
+import { AssetProxyId } from '@0x/types';
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import * as _ from 'lodash';
+import { Dispatch } from 'redux';
+import { oc } from 'ts-optchain';
+
+import { Action, actions } from '../redux/actions';
+import { State } from '../redux/reducer';
+import { AffiliateInfo, ERC20Asset } from '../types';
+import { assetUtils } from '../util/asset';
+
+import { errorFlasher } from './error_flasher';
+
+export const updateBuyQuoteOrFlashErrorAsync = async (
```

Comment:
Need to update to use https://github.com/0xProject/0x-monorepo/blob/development/packages/instant/src/util/buy_quote_updater.ts instead